packages/notebook/src/notebookfooter.ts

```diff
@@ -51,9 +53,10 @@ export class NotebookFooter extends Widget {
   }
 
   /**
-   * On arrow up key pressed (keydown keyboard event), blur the footer and switch to command mode.
+   * When an event (such as a keydown keyboard event) occurs that triggers selection of the widget above the footer,
+   * blur the footer and switch to command mode.
    */
-  protected onArrowUp(): void {
+  public onSelectAbove(): void {
```

Comment:
I think it should have a guard checking if `this.node` is focused (and if it is not, short-circuiting). This can be done comparing node with `document.activeElement`.

Reply:
Rationale is to avoid spurious switching to command mode as the setter has side effects.